docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p> </p><ul><li>*Support&nbsp;dimension* → The core&nbsp;dimensionality&nbsp;of a distribution</li></ul><p> </p><p>Maybe add a running example for these, will make it much clearer immediately. Like MvNormal 3x3, batched 4 times. Support=2, batched=4, implied=3...</p><h3><br></h3><h3>"When prototyping implied dimensions are convenient</h3><p> </p><p>"</p><h3>-&gt; When prototyping, implied dimensions are convenient</h3><p> </p><p>I would also mention that this document is more for advanced users. Most users should be fine with just knowing how dims work and use that as best practice. Ideally we'd have a separate beginner tutorial just on that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Maybe also rephrase:</p><ul><li>*Support dimensions* → Dimensions that can never be taken away from the distribution.</li></ul>

Reply:
<p>I haven’t finished the full notebook yet so maybe you talk about this later, but maybe we could also mention that the support dimensions are consumed by the elementwise  logp</p>